suite-native/device-manager/src/components/DeviceManagerModal.tsx

```diff
@@ -28,6 +28,7 @@ const deviceManagerModalWrapperStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.backgroundSurfaceElevation0,
     borderBottomLeftRadius: MANAGER_MODAL_BOTTOM_RADIUS,
     borderBottomRightRadius: MANAGER_MODAL_BOTTOM_RADIUS,
+    maxHeight: '80%',
```

Comment:
why 80%? clarify this with a comment please.

Reply:
10e7175793